flocker/node/agents/blockdevice.py

```diff
@@ -1232,6 +1228,36 @@ class _SyncToThreadedAsyncAPIAdapter(PClass):
     _threadpool = field()
 
 
+CALL_LIST_VOLUMES = MessageType(
+    u"flocker:node:agents:blockdevice:list_volumes",
+    [OPERATION, COUNT],
+    u"list_volumes called.",)
+
+
+def log_list_volumes(function):
```

Comment:
:grey_question: maybe unit test this? We have tools to unit test eliot logs, so hopefully it won't be that hard?

Perhaps this isn't that important until we have something consuming the logs, and is only as important as that consumer is.